forta-agents-linked/README.md

```diff
@@ -0,0 +1,24 @@
+# Forta Agent Assigned
+
+## Description
+
+Detects if a FORTA Agent has been assigned in a predefined threshold
+
+## Supported Chains
+
+- Polygon
+
+## Alerts
+
+- FORTA-AGENT-ASSIGNED
```

Comment:
We should change `agent` with `bot` everywhere `agent` is used

Reply:
Done